querybook/server/logic/elasticsearch.py

```diff
@@ -150,51 +159,66 @@ def _get_adhoc_query_executions_iter(batch_size=1000, session=None):
 
 
 @with_session
-def get_query_executions_iter(batch_size=1000, session=None):
-    yield from _get_query_cell_executions_iter(batch_size=batch_size, session=session)
-    yield from _get_adhoc_query_executions_iter(batch_size=batch_size, session=session)
+def get_query_executions_iter(batch_size=1000, fields=None, session=None):
+    yield from _get_query_cell_executions_iter(
+        batch_size=batch_size, fields=fields, session=session
+    )
+    yield from _get_adhoc_query_executions_iter(
+        batch_size=batch_size, fields=fields, session=session
+    )
 
 
 @with_session
-def query_execution_to_es(query_execution, data_cell=None, session=None):
+def _get_datadoc_editors(datadoc, session=None):
```

Comment:
remove the `@with_session` since session is "required" (datadoc must be bound to session)